core/src/main/java/bio/terra/pearl/core/service/export/EnrolleeImportService.java

```diff
@@ -547,6 +548,18 @@ protected SurveyResponse importSurveyResponse(UUID portalId, SurveyFormatter for
                     new ResponsibleEntity(DataAuditInfo.systemProcessName(getClass(), "handleSurveyPublished.assignToExistingEnrollees")));
             relatedTask = tasks.getFirst();
         }
+
+        // make sure the task reflects created and completion status
+        // so that recurrences are properly scheduled
+        String completedAtKey = formatter.getModuleName() + ExportFormatUtils.COLUMN_NAME_DELIMITER + "completedAt";
+        String createdAtKey = formatter.getModuleName() + ExportFormatUtils.COLUMN_NAME_DELIMITER + "createdAt";
+        if (enrolleeMap.containsKey(completedAtKey)) {
+            relatedTask.setCompletedAt(ExportFormatUtils.importInstant(enrolleeMap.get(completedAtKey)));
+        }
+        if (enrolleeMap.containsKey(createdAtKey)) {
+            relatedTask.setCreatedAt(ExportFormatUtils.importInstant(enrolleeMap.get(createdAtKey)));
+        }
```

Comment:
I needed to modify the created/found task because tasks are not exported/imported - which makes sense but I did find mildly surprising. Especially with the complexity of recurring tasks, would it make sense to start exporting tasks separately from exporting survey responses?

This would also make this PR a bit simpler, because I had to add `completedAt` as a field on survey response because it isn't exported/imported by default. If tasks were a separate concept, then that extra linkage wouldn't be needed.

Reply:
To be clear this means you'll update SurveyFormatter to be extends ModuleFormatter<SurveyResponseWithTask, ItemFormatter<SurveyResponseWithTask>> ...